1주차/10828.py

```diff
@@ -0,0 +1,27 @@
+# 스택 = last in first out
+import sys  ## 시간 초과떠서 readline 추가
+input = sys.stdin.readline
+stk = []  # 리스트로 스택 변수 만들기
+N = int(input())  # 명령 개수 받기
+for i in range(N):
+    order = input().split()  # 띄어쓰기 기준으로 리스트 생성
+    if order[0] == "push":
+        stk.append(order[1])
+    elif order[0] == "pop":
+        if len(stk) == 0:
+            print(-1)
```

Comment:
```suggestion
        if not stk:
            print(-1)
```
빈 리스트는 False로 취급되니 이렇게 쓸 수도 있어요